kubemarine/jinja.py

```diff
@@ -40,6 +40,10 @@ def _precompile(filter_: str, struct: str) -> str:
     env.filters['isipv4'] = lambda ip: ":" not in _precompile('isipv4', ip)
     env.filters['minorversion'] = lambda version: ".".join(_precompile('minorversion', version).split('.')[0:2])
     env.filters['majorversion'] = lambda version: _precompile('majorversion', version).split('.')[0]
+
+    env.tests['is_balancer'] = lambda node: 'balancer' in node['roles']
+    env.tests['is_remove_node'] = lambda node: 'remove_node' in node['roles']
```

Comment:
I suggest more generic
```suggestion
    env.tests['has_role'] = lambda node, role: role in node['roles']
```
And corresponding usage `{% if nodes | select("has_role", "balancer") | reject("has_role", "remove_node") | first %}`